ui/app/css/index.scss

```diff
@@ -4,7 +4,9 @@
   They are included first because they will be used to replace bad variable names in itcss
   prior to it being fully removed from the system.
 */
-@import './variables/index';
+@import './design-system/index';
+@import './reset.scss';
```

Comment:
Shouldn't the CSS reset come first, before anything else? It _really_ shouldn't be overriding any of our styles, and putting it first is the best way to guarantee that.

Reply:
😅 you're absolutely right, i've no idea why it isn't the first thing.

Reply:
resolved. Thanks @Gudahtt